acceptance/bundle/validate/include_locations/output.prod.json

```diff
@@ -0,0 +1,93 @@
+{
+  "files": [
+    "databricks.yml",
+    "override.yml",
+    "resources/job.yml"
+  ],
+  "locations": {
+    "bundle": [
+      [
+        0,
```

Comment:
I wonder if we need to indent JSON in this case - this works against using compact notation in the first place.

For tests we can pretty-print it with jq but for consumption in IDE, non-indent would work better.

Reply:
This is the default behavior of the indenting JSON marshaller in Go. It first serializes as JSON and then does another full pass over the returned buffer to indent it.